activejob/lib/active_job/enqueuing.rb

```diff
@@ -23,7 +23,7 @@ def perform_all_later(*jobs)
             adapter_jobs.each do |job|
               job.successfully_enqueued = false
               if job.scheduled_at
-                queue_adapter.enqueue_at(job, job.scheduled_at)
+                queue_adapter.enqueue_at(job, job.scheduled_at_time.to_f)
```

Comment:
How is a private method being allowed to be called from the instance? We have a public version as well?

Reply:
Good catch. That shouldn't work. I'll put a test on that.